connector_salesforce/salesforce_opportunity/common.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Nicolas Bessi
+#    Copyright 2014 Camptocamp SA
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import models, fields
+from ..unit.binder import SalesforceBinder
+
+
+class SalesforceOpportunity(models.Model):
+    _inherit = 'salesforce.binding'
+    _inherits = {'sale.order': 'openerp_id'}
+    _name = 'connector.salesforce.opportunity'
+    _description = 'Import SF Opportunity into sale.order model'
+
+    openerp_id = fields.Many2one('sale.order',
```

Comment:
this code is used in production, I'd really like to not change the database layout.